crates/vm/levm/src/call_frame.rs

```diff
@@ -157,18 +157,19 @@ impl CallFrame {
     }
 
     pub fn next_opcode(&self) -> Opcode {
-        match self.bytecode.get(self.pc).copied().map(Opcode::from) {
-            Some(opcode) => opcode,
-            None => Opcode::STOP,
-        }
+        self.bytecode
+            .get(self.pc)
+            .copied()
+            .map(Opcode::from)
+            .unwrap_or(Opcode::STOP)
     }
 
     pub fn increment_pc_by(&mut self, count: usize) -> Result<(), VMError> {
         self.pc = self.pc.checked_add(count).ok_or(InternalError::Overflow)?;
         Ok(())
     }
 
-    pub fn pc(&self) -> usize {
+    pub const fn pc(&self) -> usize {
```

Comment:
Does this have any practical effect? Note `const` doesn't automatically make things evaluate in compile-time, it only means it *can* be done, but it's only guaranteed to do so in `const` contexts.